packages/taiko-client/driver/preconf_blocks/api.go

```diff
@@ -307,10 +317,20 @@ func (s *PreconfBlockAPIServer) GetStatus(c echo.Context) error {
 	s.lookaheadMutex.Lock()
 	defer s.lookaheadMutex.Unlock()
 
+	endOfSequencingMarkerReceived := false
+
+	if s.rpc.L1Beacon == nil {
+		_, ok := s.sequencingEndedForEpoch.Get(s.rpc.L1Beacon.CurrentEpoch())
```

Comment:
Shall we also allow the current preconfer in his epoch to query the marker of the previous epoch?

Reply:
Not sure we need to do that - he has already started sequencing because of the `handoverSlots`. so the `lookahead.NextOperator` takes over at `slotsPerEpoch - handoverSlots`.